server/src/main/java/keywhiz/auth/ldap/LdapAuthenticatorFactory.java

```diff
@@ -71,7 +71,9 @@ public LdapLookupConfig getLookup() {
     return lookup;
   }
 
-  @Override public Authenticator<BasicCredentials, User> build(DBI dbi) {
+  // It's unfortunate that the LDAP code takes a useless UserDAO parameter. We could perhaps
```

Comment:
:+1: agreed.  It'd be nice to change this to a TODO comment so it's easy to find in a search.